src/HomeWork/hw1/Task_1_2.java

```diff
@@ -0,0 +1,42 @@
+package HomeWork.hw1;
+
+/*
+    1.2 Нарисовать ромб с звёздочек заданной величины.
+    Пример: величина 3, значит максимальное кол-во звёзд в ряду будет 3.
+  *
+ ***
+*****
+ ***
+  *
+ */
+
+public class Task_1_2 {
+
+
+    public void drawStar(int n){
```

Comment:
drawStar name not really corresponds to what method performs , name it drawRhombus, or add comment that you consider rhombus a start